### PR TITLE
[WIP] Add "Not Schedulable" to NodeStatus on nodes list page

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Only WIP because I have not been able to test locally.  My dev console currently will not run, but I'll verify this soon as possible tomorrow.

Fixes [Bugzilla 1608742](https://bugzilla.redhat.com/show_bug.cgi?id=1608742)

### CODE REVIEW

File: frontend/public/components/node.tsx [hunk 1]
```diff
@@ -54,7 +54,15 @@ const HeaderSearch = props => <ListHeader>
   <ColHead {...props} className="col-md-2 col-sm-3 hidden-xs" sortField="status.addresses">Node Addresses</ColHead>
 </ListHeader>;
 
-const NodeStatus = ({node}) => isNodeReady(node) ? <span className="node-ready"><i className="fa fa-check"></i> Ready</span> : <span className="node-not-ready"><i className="fa fa-minus-circle"></i> Not Ready</span>;
+const NodeStatus = ({node}) => {
+  if (isNodeReady(node)) {
+    return <span className="node-ready"><i className="fa fa-check"></i> Ready</span>;
+  }
+  if (!isNodeSchedulable(node)) {
```
Comment (spadgett) [inline frontend/public/components/node.tsx]: It looks like it will short-circuit before it gets here if the node is ready, but not schedulable.

Comment (benjaminapetersen) [reply frontend/public/components/node.tsx]: Prob the latter seems sufficient?

**Comment by benjaminapetersen** [comment]
Ok, updated to:

screen shot 2018-07-27 at 12 06 36 pm
screen shot 2018-07-27 at 12 06 51 pm

Open to different colors/icons/label text if anyone has a strong pref.
@tlwu2013 @rhamilto @sg00dwin

**Comment by benjaminapetersen** [comment]
CLI shows both `Ready, Scheduling Disabled`:

screen shot 2018-07-27 at 3 51 00 pm

This is accurate so is probably better.  It just looks a little ugly with the overflow.

**Comment by spadgett** [comment]
If you want to see the CLI logic, it's here:

https://github.com/kubernetes/kubernetes/blob/v1.12.0-alpha.0/pkg/printers/internalversion/printers.go#L1106

**Comment by rhamilto** [comment]
>  It just looks a little ugly with the overflow.

Overflow is fixable.  See https://github.com/openshift/console/blob/3043355b3c8eca5e6f28dca454c75793b12da404/frontend/public/components/_icon-and-text.scss

**Comment by spadgett** [comment]
If we're going to show both, we might as well stack them since it will wrap anyway.

I'd probably just take the hourglass icon off scheduling disabled. I'm not sure it fits (although I know we use that for pending routes, etc.)

**Comment by benjaminapetersen** [comment]
I can leave "✔ready", and put "scheduling disabled" below (stacked)

**Comment by benjaminapetersen** [comment]
Rebased, updated to stacked without an icon, doesn't look fantastic:

screen shot 2018-07-31 at 3 50 57 pm

But it might be fine as this table isn't super responsive yet anyway:

screen shot 2018-07-31 at 3 52 11 pm

File: frontend/public/components/node.tsx [hunk 2]
```diff
@@ -54,7 +54,22 @@ const HeaderSearch = props => <ListHeader>
   <ColHead {...props} className="col-md-2 col-sm-3 hidden-xs" sortField="status.addresses">Node Addresses</ColHead>
 </ListHeader>;
 
-const NodeStatus = ({node}) => isNodeReady(node) ? <span className="node-ready"><i className="fa fa-check"></i> Ready</span> : <span className="node-not-ready"><i className="fa fa-minus-circle"></i> Not Ready</span>;
+const NodeStatus = ({node}) => {
+  if (isNodeReady(node)) {
+    if (!isNodeSchedulable(node)) {
+      return <React.Fragment>
+        <span className="node-ready co-icon-and-text"><i className="fa fa-check co-icon-and-text__icon" aria-hidden="true"></i> Ready</span>
+        <span className="text-muted co-icon-and-text">Scheduling Disabled</span>
```
Comment (rhamilto) [inline frontend/public/components/node.tsx]: Semantically this isn't an icon and text.

**Comment by spadgett** [comment]
/close

Closing since we've recently reworked the nodes list for machines.

**Pull request closed without merging**